cms/src/main/java/uk/nhs/digital/apispecs/model/SpecificationSyncData.java

```diff
@@ -0,0 +1,111 @@
+package uk.nhs.digital.apispecs.model;
+
+import java.time.Instant;
+import java.util.regex.Pattern;
+
+public class SpecificationSyncData {
+
+    private static final Pattern VERSION_FIELD_PATTERN = Pattern.compile("\"version\"\\s*:\\s*\"[^\"]+\"");
+
+    private final ApiSpecificationDocument localSpec;
+    private final OpenApiSpecification remoteSpec;
+    private Boolean specContentChanged;
```

Comment:
I have to say no to that: I need this to be nullable so that I can easily check whether it was ever deliberately populated in `specContentChanged`. If it were a primitive, it would be automatically initialised with `false` and I'd have to use more elaborate constructs to store the info whether it was previously calculated or not.